components/library/orc/history

```diff
@@ -0,0 +1 @@
+system/library/orc@0.5.11,5.11-2022.0.0.2 library/orc
```

Comment:
Is this sufficient to let IPS see the dependency on system/library/orc as being satisfied?

Reply:
Yes.